packages/ckeditor5-dev-changelog/src/generatechangelog.ts

```diff
@@ -40,12 +40,13 @@ export async function generateChangelog( {
 	date = format( new Date(), 'yyyy-MM-dd' ) as RawDateString,
 	changesetsDirectory = CHANGESET_DIRECTORY,
 	skipLinks = false,
-	singlePackage = false
-}: RepositoryConfig & GenerateChangelog ): Promise<void> {
+	singlePackage = false,
+	returnChangelog = false
```

Comment:
```suggestion
	noWrite = false
```

If `noWrite=true`, return instead of saving. Other proposal: `emitOnly`.